web/Controllers/AbstractController.cs

```diff
@@ -11,4 +15,19 @@ protected BricksAndHeartsUser GetCurrentUser()
 
         return (BricksAndHeartsUser)User.Identity;
     }
+
+    protected void FlashRequestSuccess(ILogger logger, BricksAndHeartsUser user, string requestAction)
+    {
+        FlashMessage(logger,
+            ($"Successfully {requestAction} for user {user.Id}",
+                "success",
+                $"Successfully {requestAction}"));
+    }
+
+    protected void FlashMessage(ILogger logger, (string logInfo, string flashtype, string flashmessage) flash)
+    {
+        logger.LogInformation(flash.logInfo);

```

Comment:
I think having the log statement here is probably overkill, but doesn't really matter, we can always take it out later if we don't find it useful